alpha-build-machinery/make/targets/golang/verify-update.mk

```diff
@@ -26,6 +28,27 @@ verify-golint:
 	$(GOLINT) $(GO_PACKAGES)
 .PHONY: verify-govet
 
+verify-sec: update-gosec
+ifeq (, $(shell which $(GOSEC) 2>/dev/null))
+	$(GOSEC_GOPATH)/bin/$(GOSEC) \
+		-severity $(GOSEC_SEVERITY) -confidence $(GOSEC_CONFIDENCE) \
+		-exclude $(GOSEC_EXCLUDE) \
+		-quiet $(GO_PACKAGES)
+else
+	$(GOSEC) \
+		-severity $(GOSEC_SEVERITY) -confidence $(GOSEC_CONFIDENCE) \
+		-exclude $(GOSEC_EXCLUDE) \
+		-quiet $(GO_PACKAGES)
+endif
+.PHONY: verify-sec
+
+update-gosec:
+ifeq (, $(shell which $(GOSEC) 2>/dev/null))
+	GOPATH=$(GOSEC_GOPATH) GOBIN=$(GOSEC_GOPATH)/bin GO111MODULE=on go get github.com/securego/gosec/cmd/gosec@4b59c948083cd711b6a8aac8f32721b164899f57
```

Comment:
not sure why this should be selecting GO111MODULE for the user?

Reply:
In order for us to get a pinned version. Which addresses your other comment.

Reply:
So we get a pinner version